components/molecule/rating/README.md

```diff
@@ -33,6 +33,24 @@ import MoleculeRating, {MoleculeRatingSizes} from '@s-ui/react-molecule-rating'
 <MoleculeRating value={4} label="25 opiniones" size={MoleculeRatingSizes.MEDIUM} link href="https://www.adevinta.com/"/>        
 ```
 
+### With CUSTOM icons
+
+```js
+import {IconStarFilled, IconStarHalfFilled, IconStarOutline} from './Icons'
+
+const customPropsStar = {
+  IconStarFilled,
+  IconStarHalfFilled,
+  IconStarOutline
```

Comment:
It should be "IconStarEmpty"

Reply:
MoleculeRatingStar has IconStarEmpty, IconStarFilled and IconStarHalfFilled

Reply:
You're right. Fixed!